packages/lpa-questionnaire-web-app/src/validators/development-plan.js

```diff
@@ -0,0 +1,27 @@
+const { body } = require('express-validator');
+
+const rules = () => {
+  const developmentPlanRef = 'has-plan-submitted';
+  const developmentPlanValues = ['yes', 'no'];
+  const developmentPlanTextRef = 'plan-changes-text';
+
+  return [
+    body(developmentPlanRef)
+      .notEmpty()
+      .withMessage(
+        'Has any Development Plan Document or Neighbourhood Plan relevant to this appeal been submitted for examination (or in the case of a Neighbourhood Plan, been examined and is awaiting a referendum)?'
```

Comment:
This is not the correct error. Should be 'Select yes if there is a relevant Development Plan or Neighbourhood Plan'

Reply:
Corrected